### DISCUSSION TITLE
Recurring Event Days That Are Completed Still Showing from Shortcode

### DISCUSSION THREAD

**Discussion by AllenFreeman** [liedekef/events-made-easy#248] [Q&A]
I'm using three different eme_events shortcodes on the front page of https://northernohio.golf to show three different category calendars. In the last few weeks, I'm suddenly having recurring event days that are completed stay showing on the list of events  (i.e. a 3-day event will continue to show the Saturday event listing even though it's actually Sunday or Monday when I'm looking at it).

Any ideas on things I can check to get these recurring event days to no longer display in the shortcode once they are in the past?

Thanks!

**Comment by liedekef** [comment]
It depends on your event definition. EME doesn't care about recurrence during the list of events (not by default anyway), but just looks at the start/end date of an event. I'm guessing you're using a recurrent event and also specified the event lasts 3 days or so ... 
Recurring events are not to be confused with an event lasting more than one day, so check that.

**Reply by AllenFreeman** [reply]
Please help me understand how a 3 consecutive day event at the same location should be set up, because yes, I think I'm recurring/specifying.

When I create an event for a 3 day tournament at the same golf course, I click the checkbox for "Check if your event happens more than once." then I set the start and end dates for the three days. I also go up above that and change the number to 3 in the field for "Event duration (in days)".

Should I be doing that differently?

**Reply by AllenFreeman** [reply]
I think I figured it out. Set the start/end dates, then don't put anything into the field for the number of days. Right?

**Reply by liedekef** [reply]
Just use start and end date, don't use "Check if your event happens more than once." if you don't need it. A recurring event means that the same event will be repeated per the recurrence definition (like you will see in the EME backend , in the list of events).
